wlc/__init__.py

```diff
@@ -18,13 +18,19 @@
 #
 """Weblate API client library."""
 
+import json
+import logging
 from copy import copy
-from urllib.parse import urlencode, urlparse
 
 import dateutil.parser
 import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from six.moves.urllib.parse import urlencode, urlparse
```

Comment:
There is no need to reintroduce six dependency. Seems like something went wrong when you merged upstream changes...